Okta.Xamarin/Okta.Xamarin/OktaXamarinUserAgent.cs

```diff
@@ -0,0 +1,15 @@
+﻿// <copyright file="OktaXamarinUserAgent.cs" company="Okta, Inc">
+// Copyright (c) 2020 - present Okta, Inc. All rights reserved.
+// Licensed under the Apache 2.0 license. See the LICENSE file in the project root for full license information.
+// </copyright>
+
+using System;
+using System.Reflection;
+
+namespace Okta.Xamarin
+{
+	public class OktaXamarinUserAgent
+    {
+        public static string Value => new Lazy<string>(() => $"Okta-Xamarin-Sdk/{Assembly.GetExecutingAssembly().GetName().Version}").Value;
```

Comment:
Also this needs to be changed the same as in PR #35

Reply:
That change will be picked up when 35 is merged.

Reply:
Or I could change it here and just merge this PR instead of 35.  Small enough that it shouldn't be a problem either way.